Bottom: 5887c383f13a1270b3bdcdf20d7142cab2e8de29
Top:    d6817c6fb6662a96947ac85a3d21150ac3f5f868
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-20 14:11:55 +0530

Refresh of sched-fair-tune-task-wake-up

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index f4cd27593a62..017ac3e38d87 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5888,7 +5888,7 @@ static int select_idle_cpu(struct task_struct *p, struct sched_domain *sd, int t
 #ifdef CONFIG_SCHED_SMT
 
 /* Define non-idle CPU as the one with the utilization >= 12.5% */
-#define is_cpu_non_idle(util) ((util) > (100 >> 3))
+#define dawdling_cpu(util) ((util) > (100 >> 3))
 
 /*
  * Classify small background tasks with higher latency_nice value for task
@@ -5932,7 +5932,7 @@ static int select_non_idle_core(struct task_struct *p, int prev_cpu)
 				non_idle_cpu_count++;
 				if (cpu_overutilized(sibling))
 					overutil_cpu_count++;
-				if (is_cpu_non_idle(cpu_util(sibling)))
+				if (dawdling_cpu(cpu_util(sibling)))
 					busy_cpu_count++;
 			}
 		}
